src/test/java/org/skriptlang/skript/test/tests/lang/DefaultExpressionErrorTest.java

```diff
@@ -0,0 +1,114 @@
+package org.skriptlang.skript.test.tests.lang;
+
+import ch.njol.skript.lang.DefaultExpressionError;
+import ch.njol.skript.test.runner.SkriptJUnitTest;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+
+public class DefaultExpressionErrorTest extends SkriptJUnitTest {
+
+	@Test
+	public void test_not_found() {
+		Assert.assertEquals(
+			DefaultExpressionError.NOT_FOUND.getError(List.of("itemtype"), "itemtype"),
+			"The class 'itemtype' does not provide a default expression. Either allow null (with %-itemtype%) " +
+				"or make it mandatory [pattern: itemtype]"
+		);
+
+		Assert.assertEquals(
+			DefaultExpressionError.NOT_FOUND.getError(List.of("itemtype", "entity"), "itemtype/entity"),
+			"The classes 'itemtype and entity' do not provide a default expression. Either allow null (with %-itemtype/entity%) " +
+				"or make it mandatory [pattern: itemtype/entity]"
+		);
+
+		Assert.assertEquals(
+			DefaultExpressionError.NOT_FOUND.getError(List.of("itemtype", "entity", "object"), "itemtype/entity/object"),
+			"The classes 'itemtype, entity and object' do not provide a default expression. Either allow null " +
```

Comment:
```suggestion
			"The classes 'itemtype, entity, and object' do not provide a default expression. Either allow null " +
```
Prefer using the oxford comma to avoid any possible ambiguity